idd/Energy+.idd.in

```diff
@@ -20983,188 +20983,233 @@ ZoneContaminantSourceAndSink:Generic:DepositionRateSink,
 \group Daylighting
 
 Daylighting:Controls,
-       \min-fields 19
-       \memo Dimming of overhead electric lighting is determined from
-       \memo interior daylight illuminance calculated at one or two reference points.
-       \memo reference points are given in coordinates specified in the GlobalGeometryRules object
-       \memo Daylighting Reference Point CoordinateSystem field
+       \extensible:3 - repeat last three fields remembering to remove ; from "inner" fields.
+       \min-fields 16
+       \memo Dimming of overhead electric lighting is determined from each reference point.
        \memo Glare from daylighting is also calculated.
-  A1 , \field Zone Name
+  A1,  \field Name
+       \required-field
+       \type alpha
+  A2,  \field Zone Name
        \required-field
        \type object-list
        \object-list ZoneNames
-  N1 , \field Total Daylighting Reference Points
+  A3,  \field Daylighting Method
+       \required-field
+       \type choice
+       \key SplitFlux
+       \key DElight
+       \default SplitFlux
+  A4,  \field Availability Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+  A5,  \field Lighting Control Type
+       \required-field
+       \type integer
+       \type choice
+       \key Continuous
+       \key Stepped
+       \key ContinuousOff
+       \default Continuous
+  N1,  \field Minimum Input Power Fraction for Continuous or ContinuousOff Dimming Control
+       \type real
+       \minimum 0.0
+       \maximum 0.6
+       \default 0.3
+  N2,  \field Minimum Light Output Fraction for Continuous or ContinuousOff Dimming Control
+       \type real
+       \minimum 0.0
+       \maximum 0.6
+       \default 0.2
+  N3,  \field Number of Stepped Control Steps
+       \note for Stepped Lighting Control Type, this field cannot be zero.
        \type integer
-       \minimum 1
-       \maximum 2
        \default 1
```

Comment:
It is funny that the \minimum and \maximum look like they have been removed from the "Number of Stepped Control Steps", those lines of the IDD are actually originally from the "Total Daylighting Reference Points". I will add a \minimum 1